guide/source/program_administrator/data_validation.rst

```diff
@@ -72,9 +74,9 @@ can significantly reduce your labor requirements for reviewing submitted files
 as you can focus your labor on probable errors or problem areas.  In some cases,
 you may be able to get to auto-approval on a select number of projects.
 
-Over time you can add ever more sophisticated systems. What is most important to
+[Over time you can add ever more sophisticated systems. What is most important to
 remember is that, if you launch with a very complex validation system, there is
 a high likelihood that many of the initial project submissions will fail as the
-market is still adapting to the new program environment. If you take a phased
+market is still adapting to the new program environment.--> THIS SHOULD PROBABLY BE HIGHLIGHTED OR INLCUDED IN A CALL-OUT BOX.  MAYBE IT WOULD ALSO GO INTO A CHEAT-SHEET COMPANION TO THE GUIDE THAT DESCRIBES QUICKLY WHAT MAJOR THINGS A PROGRAM SHOLD LOOK AT/FOR WHEN CONSIDERING HPXML. If you take a phased
```

Comment:
@Gavinhastings This can be done with the following reStructuredText directives. I can help you with this if you need.
```
.. note::
    
    highlighted text here.

or 

.. warning::

    really highlighted text here.
```